src/libwalletqt/Wallet.cpp

```diff
@@ -215,6 +215,16 @@ PendingTransaction *Wallet::createTransaction(const QString &dst_addr, const QSt
     return result;
 }
 
+PendingTransaction *Wallet::createTransactionAll(const QString &dst_addr, const QString &payment_id,
+                                                 quint32 mixin_count, PendingTransaction::Priority priority)
+{
+    Bitmonero::PendingTransaction * ptImpl = m_walletImpl->createTransaction(
+                dst_addr.toStdString(), payment_id.toStdString(), boost::optional<uint64_t>(), mixin_count,
```

Comment:
Can it be done without boost? Something like passing 0 here? Initially idea was to not introduce anything except stl into libwallet_api interface. boost currently isn't compile time dependency for GUI (only libs), so at least Windows build will break. Otherwise boost include path needs to be added to monero-core.pro